docs/deploying/BBSSignatures.md

```diff
@@ -0,0 +1,27 @@
+# BBS+ Signatures Support
+
+ACA-Py has supported BBS+ Signatures for some time. However, the dependency that is used (`bbs`) does not support ARM architecture, and its inclusion in the default ACA-Py artifacts mean that developers using ARM-based hardware (such as Apple M1 Macs or later) cannot run ACA-Py "out-of-the-box". We feel that providing a better developer experience by supporting ARM architecture is more important than BBS+ Signature support at this time. As such, we have removed the BBS+ dependency from the base ACA-Py artifacts, and made it an add-on, that those using ACA-Py with BBS+ must take extra steps to build their own artifacts. This file describes how to do those extra steps.
```

Comment:
A couple of tweaks here, pretty please:

- “BBS” and “BBS+” are two different signature schemes.  I think we should just use “BBS” throughout, even if it is not 100% correct in all places.  I finally heard the history of where the “+” came from, and why it has gone away again — fun stuff.
- "does not support ARM architecture” -- should be “the ARM…”, same with “…supporting the ARM architecture”
- Please remove the second and third commas in the phrase: `As such, we have removed the BBS+ dependency from the base ACA-Py artifacts, and made it an add-on, that those using ACA-Py with BBS+ must take extra steps to …`